src/components/deposit/Congratulations.js

```diff
@@ -60,11 +54,13 @@ const Congratulations = ({ depositAddress, lotInSatoshis, signerFeeInSatoshis, c
   </div>
 }
 
-const mapStateToProps = (state, ownProps) => {
+const mapStateToProps = (state) => {
+  const { depositAddress, lotInSatoshis, signerFeeInSatoshis } = state.deposit
+  const mintedSatoshis = lotInSatoshis.sub(signerFeeInSatoshis)
```

Comment:
What's the thinking on moving this and the formatting call in the previous file to `mapStateToProps`?

Reply:
It feels a little cleaner to me to move the data massaging out of the component itself. We used this type of pattern with the [reselect](https://www.npmjs.com/package/reselect) package in the Fold web app, but I thought maybe using reselect itself maybe a little heavy handed. Though, one of the benefits of reselect is memoization...I wasn't sure if that benefit outweighed adding another package in our use case, so I settled on creating the formatting util and moving this stuff into the `mapStateToProps`.

Reply:
Got it! Mostly was looking for reasoning, I don't have a strong opinion here (yet? :laughing:).